internal/config/config.go

```diff
@@ -164,3 +167,9 @@ type saveError struct {
 func (e saveError) Error() string {
 	return fmt.Sprintf("unable to save the configuration: %v", e.err)
 }
+
+// GetV1UnsupportedError returns the error to inform that V1 is no longer supported
+func GetV1UnsupportedError() error {
+	return fmt.Errorf(noticeColor, "The v1 projects are deprecated since Feb 1, 2020 and are no longer supported.\n"+
```

Comment:
This statement is incorrect since v1 projects have been deprecated for a long time, not just since feb. Instead of stating a date, just say they're no longer supported.